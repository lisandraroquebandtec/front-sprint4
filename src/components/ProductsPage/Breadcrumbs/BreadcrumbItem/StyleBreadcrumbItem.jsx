import styled from "styled-components";

export const Item = styled.li`
  font-size: 0.8125rem;
  font-weight: 600;
  color: #585858;
  margin-right: 5px;
  &:not(:last-child):after {
    content: ' / ';
  }
  &:last-child {
    color: #261e1e;
  }
  &:not(:last-child):hover {
    text-decoration: underline;
  }
  @media (max-width: 1200px) {
    display: none;
    &:last-child {
      display: block;
      text-align: center;
      font-size: 1.4375rem;
      font-weight: 400;
      text-transform: uppercase;
    }
  }
`;

export const BreadcrumbLink = styled.a`
  /* font-size: 0.8125rem;
  font-weight: 600;
  color: #585858;
  text-decoration: none;

  @media (max-width: 1200px) {
    display: none;
  }
  &:not(:last-child):hover {
    text-decoration: underline;
  } */
`;

// export const Separator = styled.li`
//   margin-left: 8px;
//   margin-right: 8px;
//   user-select: none;
// `;
