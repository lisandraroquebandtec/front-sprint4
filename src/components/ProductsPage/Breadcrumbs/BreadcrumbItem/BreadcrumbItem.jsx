import { Item, BreadcrumbLink} from "./StyleBreadcrumbItem";

function BreadcrumbItem(props) {
  return (
    
      <Item>
        <BreadcrumbLink href={props.link}>{props.breadcrumbs}</BreadcrumbLink>
      </Item>
      /* <Separator>/</Separator> */
   
  );
}
export default BreadcrumbItem;

