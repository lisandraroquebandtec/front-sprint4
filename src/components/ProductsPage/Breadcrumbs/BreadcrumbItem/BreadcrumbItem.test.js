import React from 'react';
import { render} from '@testing-library/react';
import BreadcrumbItem from './BreadcrumbItem';

describe('O componente de item do breadcrumbs,', () =>{
  it('o snapshot do componente deve permanecer sempre o mesmo', ()=>{
    const {container} = render(
    <BreadcrumbItem
        link='/home' name='home'
    />)
    expect(container.firstChild).toMatchSnapshot();
  })
})