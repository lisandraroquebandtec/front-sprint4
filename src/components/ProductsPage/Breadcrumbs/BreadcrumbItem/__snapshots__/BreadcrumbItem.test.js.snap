// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`O componente de item do breadcrumbs, o snapshot do componente deve permanecer sempre o mesmo 1`] = `
<li
  class="sc-bdnxRM lbqNSD"
>
  <a
    class="sc-gtsrHT eSDBZc"
    href="/home"
  />
</li>
`;
